website/src/pages/chat/index.tsx

```diff
@@ -4,33 +4,30 @@ import { getToken } from "next-auth/jwt";
 import { useTranslation } from "next-i18next";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import React from "react";
-import { ChatContextProvider } from "src/components/Chat/ChatContext";
-import { ChatSection } from "src/components/Chat/ChatSection";
-import { getChatLayout } from "src/components/Layout/ChatLayout";
+import { ChatListBase } from "src/components/Chat/ChatListBase";
+import { getDashboardLayout } from "src/components/Layout";
 import { isChatEnable } from "src/lib/isChatEnable";
 import { createInferenceClient } from "src/lib/oasst_inference_client";
-import { ModelInfo } from "src/types/Chat";
+import { GetChatsResponse } from "src/types/Chat";
 
 type ChatListProps = {
-  modelInfos: ModelInfo[];
+  chatResponse: GetChatsResponse;
 };
 
-const ChatList = ({ modelInfos }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+const ChatList = ({ chatResponse }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   const { t } = useTranslation(["chat"]);
 
   return (
     <>
       <Head>
         <title>{t("chat")}</title>
       </Head>
-      <ChatContextProvider modelInfos={modelInfos} messages={[]}>
-        <ChatSection chatId={null} />
-      </ChatContextProvider>
+      <ChatListBase chats={chatResponse} isSideBar={false} className="max-w-5xl mx-auto" />
```

Comment:
This seems to be an ugly UI IMO.

Reply:
no problem, we can create another PR and change the design.